jetty-server/src/main/java/org/eclipse/jetty/server/handler/BufferedResponseHandler.java

```diff
@@ -203,7 +203,7 @@ protected BufferedInterceptor newBufferedInterceptor(HttpChannel httpChannel, In
     {
     }
 
-    private class ArrayBufferedInterceptor implements BufferedInterceptor
+    protected class ArrayBufferedInterceptor implements BufferedInterceptor
```

Comment:
Does it work if you make the class just package-private rather than protected?

Reply:
It does if I move the test to the same package.
Done.